semgrep/semgrep/semgrep_types.py

```diff
@@ -66,6 +67,7 @@ class Language_util:
         Language.RUST: [Language.RUST.value, "Rust", "rs"],
         Language.KOTLIN: [Language.KOTLIN.value, "Kotlin", "kotlin"],
         Language.YAML: [Language.YAML.value, "Yaml"],
+        Language.SCALA: [Language.SCALA.value, "scala"],
```

Comment:
These two are the same thing :+1: 

```suggestion
        Language.SCALA: [Language.SCALA.value],
```

Reply:
oops, copy pasting without thinking

Reply:
I didn't know you could commit suggested changes. How do you suggest those changes? There's a button somewhere?

Reply:
Yup, above the comment box with the little "+" and "-":

![2021-05-21-100019_904x408_scrot](https://user-images.githubusercontent.com/1724818/119166081-72edc300-ba1b-11eb-958c-52042772514d.png)

Reply:
And you can drag the mouse over multiple lines to suggest multiple!